shared/continuations/lib/LowerRaytracingPipeline.cpp

```diff
@@ -759,6 +760,32 @@ void LowerRaytracingPipelinePassImpl::handleContinuationStackIsGlobal(
   }
 }
 
+void LowerRaytracingPipelinePassImpl::handleGetFuncAddr(Function &Func) {
+  assert(Func.arg_empty()
+         // returns i64 or i32
+         && (Func.getFunctionType()->getReturnType()->isIntegerTy(64) ||
+             Func.getFunctionType()->getReturnType()->isIntegerTy(32)));
+
+  auto Name = Func.getName();
+  bool Consumed = Name.consume_front("_AmdGetFuncAddr");
+  assert(Consumed);
+  (void)Consumed;
+
+  Constant *Addr = Mod->getFunction(Name);
+  if (!Addr)
+    report_fatal_error(Twine("Did not find function '") + Name +
+                       "' requested by _AmdGetFuncAddr");
+  Addr = ConstantExpr::getPtrToInt(Addr, Func.getReturnType());
+  for (auto &Use : make_early_inc_range(Func.uses())) {
+    if (auto *CInst = dyn_cast<CallInst>(Use.getUser())) {
+      if (CInst->isCallee(&Use)) {
```

Comment:
I think this can use `forEachCall`.

Reply:
Yeah, requires to append make_early_inc_range here as well, I'll change that.